### PR TITLE
chore(dev): update @substrate/dev

### PR DESCRIPTION
**Pull request by TarikGul**

### CODE REVIEW

**Review by lexnv — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -36,7 +36,6 @@
     "dev": "tsc-watch --onSuccess \"yarn run main\"",
     "test": "NODE_ENV=test substrate-exec-jest --detectOpenHandles",
     "test:watch": "NODE_ENV=test substrate-exec-jest --watch",
-    "test:cov": "NODE_ENV=test substrate-exec-jest --detectOpenHandles --coverage",
```
Comment (niklasad1) [inline package.json:39]: why was this removed?

Comment (TarikGul) [reply package.json:39]: Good question. We dont actually use it anywhere, but also with Jest updated, it requires us to set the `coverageProvider` to `v8` and off of the default which is babel. `v8` is also experimental at the moment and causes some side-affects for us in the CI, so I removed it.

**Review by jsdw — APPROVED** [review]
